src/components/item-info/index.js

```diff
@@ -96,57 +96,61 @@ export const ItemInfo = ({
 
   return (
     <>
-      <div className={styles.container}>
-        <div className={styles.edition}>
-          <div className={styles.inline}>
-            <p className={styles.issuer}>Issuer:&nbsp;</p>
-            <Button to={`${PATH.ISSUER}/${token_info.creators[0]}`}>
-              <Primary>{walletPreview(token_info.creators[0])}</Primary>
-            </Button>
-          </div>
-          {!feed && (
-            <div>
-              <p>
-                <span>
-                  Editions:
+      <article>
+        <footer className={styles.container}>
+          <div className={styles.edition}>
+            <div className={styles.inline}>
+              <p className={styles.issuer}>Issuer:&nbsp;</p>
+              <Button to={`${PATH.ISSUER}/${token_info.creators[0]}`}>
+                <Primary>{walletPreview(token_info.creators[0])}</Primary>
+              </Button>
+            </div>
+            {!feed && (
+              <div>
+                <p>
                   <span>
-                    {ed}/{total}
+                    Editions:
+                    <span>
+                      {ed}/{total}
+                    </span>
                   </span>
-                </span>
-              </p>
-              {false && (
-                <p>
-                  Price range: {(Number(s.xtz_per_objkt) / 1000000).toFixed(2)}-
-                  {(Number(maxPrice.xtz_per_objkt) / 1000000).toFixed(2)}
                 </p>
-              )}
+                {false && (
+                  <p>
+                    Price range: {(Number(s.xtz_per_objkt) / 1000000).toFixed(2)}-
+                    {(Number(maxPrice.xtz_per_objkt) / 1000000).toFixed(2)}
+                  </p>
+                )}
+              </div>
+            )}
+          </div>
+          {feed && (
+            <div className={styles.objktContainer}>
+              <Button to={`${PATH.OBJKT}/${token_id}`} disabled={isDetailView}>
+                <Primary>OBJKT#{token_id}</Primary>
+              </Button>
+              <div className={styles.hdaoButton}>
+                {renderHDAObutton(token_id, hDAO_balance)}
+              </div>
             </div>
           )}
-        </div>
-        {feed && (
-          <div className={styles.objktContainer}>
-            <Button to={`${PATH.OBJKT}/${token_id}`} disabled={isDetailView}>
-              <Primary>OBJKT#{token_id}</Primary>
-            </Button>
-            <div className={styles.hdaoButton}>
-              {renderHDAObutton(token_id, hDAO_balance)}
+        </footer>
+        <header className={styles.container}>
```

Comment:
Is this right? a <footer> before a <header> ?

Reply:
From the MDN Guide https://developer.mozilla.org/en-US/docs/Web/Guide/HTML/Using_HTML_sections_and_outlines

"HTML Footer Element (<footer>) defines a page footer, which typically contains copyright or legal notices and sometimes some links. In the context of a section, a footer might contain the sectioned content's publication date, license information, etc. <article>, <section>, <aside>, and <nav> can have their own <footer>. **Despite its name, the footer is not necessarily positioned at the end of a page or section.**"

Reply:
I can make it less strict if you guys want

Reply:
I took out the stricter tags in my latest commit

Reply:
I'm just not sure about the footer. As normally I would separate it from the article or section. in my head it makes sense to be a global thing like a header is, detatched from the current article being viewed. but its not the end of the world.

Reply:
i took a lot of html tags out / made everything simpler